src/js/components/FormField/FormField.js

```diff
@@ -221,13 +221,26 @@ const FormField = forwardRef(
     contents = <Box {...contentProps}>{contents}</Box>;
 
     let borderColor;
-    if (focus && !normalizedError) {
+
+    if (disabled) {
+      borderColor =
+        formField.disabled.border && formField.disabled.border.color;
+    } else if (focus && !normalizedError) {
       borderColor = 'focus';
     } else if (normalizedError) {
       borderColor = (border && border.error.color) || 'status-critical';
     } else {
       borderColor = (border && border.color) || 'border';
     }
+
+    const labelStyle = formField.label;
+
+    if (disabled) {
+      labelStyle.color = formField.disabled.label
+        ? formField.disabled.label.color
+        : labelStyle.color;
```

Comment:
Do we have a story or a test to verify backward compatibility of `formField.label` object, and color in specific?

Reply:
I took care of this one on https://github.com/grommet/grommet/pull/3998

Reply:
Thank you. Worked like a charm.